spec/Pim/Bundle/DataGeneratorBundle/Generator/ProductGeneratorSpec.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace spec\Pim\Bundle\DataGeneratorBundle\Generator;
+
+use PhpSpec\ObjectBehavior;
+use Pim\Bundle\CatalogBundle\Repository\AttributeRepositoryInterface;
+use Pim\Bundle\CatalogBundle\Repository\CategoryRepositoryInterface;
+use Pim\Bundle\CatalogBundle\Repository\ChannelRepositoryInterface;
+use Pim\Bundle\CatalogBundle\Repository\CurrencyRepositoryInterface;
+use Pim\Bundle\CatalogBundle\Repository\FamilyRepositoryInterface;
+use Pim\Bundle\CatalogBundle\Repository\LocaleRepositoryInterface;
+use Prophecy\Argument;
+
+class ProductGeneratorSpec extends ObjectBehavior
+{
+    function let(
+        FamilyRepositoryInterface $familyRepository,
+        AttributeRepositoryInterface $attributeRepository,
+        ChannelRepositoryInterface $channelRepository,
+        LocaleRepositoryInterface $localeRepository,
+        CurrencyRepositoryInterface $currencyRepository,
+        CategoryRepositoryInterface $categoryRepository
+    ) {
+        $this->beConstructedWith(
+            $familyRepository,
+            $attributeRepository,
+            $channelRepository,
+            $localeRepository,
+            $currencyRepository,
+            $categoryRepository
+        );
+    }
+
+    function it_is_initializable()
```

Comment:
- change the function name by "initializable" or "isInitializable" instead
- use camel case